tests/e2e/specs/add-paid-campaigns/step-1-accounts.test.js

```diff
@@ -244,6 +273,89 @@ test.describe( 'Set up Ads account', () => {
 			await expect(
 				page.getByRole( 'heading', { name: 'Set your budget' } )
 			).toBeVisible();
+
+			await expect(
+				page.getByRole( 'button', { name: 'Continue' } )
+			).toBeDisabled();
+
+			await expect(
+				page.getByRole( 'link', {
+					name: 'See what your ads will look like.',
+				} )
+			).toBeVisible();
+		} );
+
+		test.describe( 'Preview product ad', () => {
+			test( 'Preview product ad should be visible', async () => {
+				await expect(
+					page.getByText( 'Preview product ad' )
+				).toBeVisible();
+				await expect(
+					page.getByText(
+						"Each of your product variants will have its own ad. Previews shown here are examples and don't include all possible formats."
+					)
+				).toBeVisible();
+			} );
+
+			test( 'Change image buttons should be enabled', async () => {
+				const buttonsToChangeImage = page.locator(
+					'.gla-campaign-preview-card__moving-button'
+				);
+
+				expect( await buttonsToChangeImage.count() ).toBe( 2 );
```

Comment:
It's more recommended to use web first assertions.

```suggestion
				await expect( buttonsToChangeImage ).toHaveCount( 2 );
```